cfme/infrastructure/virtual_machines.py

```diff
@@ -544,7 +544,7 @@ def get_changes_to_fill(self, other_configuration):
         return changes
 
 
-class Vm(VM):
+class InfraVm(VM):
     """Represents a VM in CFME
```

Comment:
**QUESTION:** Shouldn't the comment change too?

Reply:
It can, but I don't think its necessary for all comment uses to specify `InfraVm` unless specifically referring to the class, here its just a generic description.  I'll update it if you say **Required** :)

Reply:
I just want it to say represents an Infrastructure VM and maybe spell out how they are different than the rest of their VM kind...not every where just here at the class description.